run/markdown-preview/editor/editor.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">

```

Comment:
I'll rename it to be consistent with the other samples.

At the time, I structured it out to be consistent with the other languages, especially since the tutorial in the docs references directory names in the steps.